.github/workflows/assets-test.yaml

```diff
@@ -79,6 +80,10 @@ jobs:
       - name: Install dependencies
         run: pip install -e $scripts_azureml_assets_dir
 
+      - name: Setup azcopy
+        shell: bash
+        run: bash ${{ scripts_dir.scripts_dir }}/install_azcopy.sh
+
```

Comment:
Please remove. azcopy won't be used during this job. It's just for checking the execution context.

Reply:
We don't test assets publish to some temp ws during PR creation?

Reply:
Awesome. It is needed for the test job below, so let's leave it there.